docs/articles/documentation/extending-testcafe/custom-reporter/creating-reporter-plugin/index.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: docs
+title: Creating Custom Reporter
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/creating-reporter-plugin/
+---
+# Creating Reporter Plugin
+
+## Generating Reporter Project
+
+First, install [Yeoman](http://yeoman.io) and `generator-testcafe-reporter` using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
```

Comment:
Is it really necessary to say about node.js? I can't imagine how someone without node.js installed (and, hence, without testcafe) will try to follow this tutorial.